modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a Subscription
+
+You can create a Subscription to connect a Service or other event sink to a Channel.
+The following procedure shows how to create a Subscription object that uses the default InMemoryChannel type Channel.
+
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
+====
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+// * If you want to use other Channel types with Subscriptions...
+// TODO: Create docs for these, link Kafka etc?
+
+.Procedure
+
+* Create a Subscription object that uses the default InMemoryChannel type Channel by entering the following:
++
+[source,yml]
+----
+$ oc  create -f - <<EOF
+apiVersion: messaging.knative.dev/v1
+kind: Subscription
+metadata:
+  name: my-subscription
```

Comment:
We should consider if we want to have "namespace" here (we do so in previous examples) ... if "namespace" is not specified, it would be created in the "current namespace", as set by "oc namespace"